pkg/volume/portworx/portworx_util.go

```diff
@@ -38,6 +41,7 @@ const (
 	pvcNamespaceLabel = "namespace"
 	pxServiceName     = "portworx-service"
 	pxDriverName      = "pxd-sched"
+	oneGig            = 1024 * 1024 * 1024
```

Comment:
use `oneGi` instead?

Reply:
Removed this and now using existing functions in https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/util.go